.github/workflows/Test.yml

```diff
@@ -0,0 +1,128 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+# Displaye name of the workflow.
+name: Test
+
+# Workflow Trigger
+# Trigger the workflow on a pull request to any branch, including master
+on:
+  pull_request:
+  # This triggers the workflow in the event there is a push directly to master
+  push:
+    branches: 
+      - master
+
+jobs:
+  # This workflow file defines only one job, "build". The steps have been made generic to run on Linux, Windows, and Mac.
+  build:
+    # Display name of the job
+    name: Express Build
+    # The OS to run the job steps on. We are using a matrix definition that defines the node versions and the OS to use, so
+    # here we just specify use the OS defined in the matrix.
+    runs-on: ${{ matrix.os }}
+    # This prevents a workflow from failing just because a specific job fails. In this case it checks the experimental tag in 
+    # the matrix
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        # Added node version 13 and 14 based off feedback from the group.
+        node-version: ["0.10", "0.12", "1.8", "2.5", "3.3", "4.9", "5.12", "6.17", "7.10", "8.17", "9.11", "10.19", "11.15", "12.16", "13.0", "14.0"]
```

Comment:
Sorry there may have been confusion on the thread about the versions. Please keep the versions the same so we keep this as a change to GH actions. Adding 14 is also more involved than adding it here, as we need to validate our deps first.

But it would be nice to format this as a version per line so as we keep adding and removing versions it merges better in git being separate lines.

Reply:
Does that mean you would like me to add back the nightly builds as well?

Reply:
@dougwilson it doesn't appear that I can break the matrix over multiple lines. It doesn't look like that is supported at this time.

Reply:
What isn't supported? Isn't this a YAML file, so just use the list notation instead of this inline notation?

Reply:
For nightly: yea, if you can. That would also double as we know how it works.

Reply:
I'll add the nightly back.  And yes, it is a YAML, so I'm thinking I may have done it wrong. I'll give the list another shot as well.

Reply:
Yep it was an error between the keyboard and chair. I've added the nightly builds back, and I've turned the matrix into a list.

Reply:
And I removed versions 13 and 14 for the matrix. At this point it should be doing exactly what the travis and appveyor builds do.